### PR TITLE
add link to helm repo to readme

### LINKED ISSUE #6: Adding the chart repository
I would really like to switch over from the old stable repository and try this one.
However, I couldn't find the right URL in the docs in order to add the repo using `helm repo add`.

### PR DESCRIPTION
**Pull request by erkannt**
closes #6

### CODE REVIEW

File: helm/oauth2-proxy/README.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 ## TL;DR;
 
 ```console
+$ helm repo add oauth2-proxy https://oauth2-proxy.github.io/manifests
 $ helm install stable/oauth2-proxy
```
Comment (JoelSpeed) [inline helm/oauth2-proxy/README.md:9]: Is it still called `stable/oauth2-proxy`? Is that what we want to have it called?

Comment (SeWieland) [reply helm/oauth2-proxy/README.md:9]: It's whatever you "add" the repo as (repo/chart). So in this case it should be
```suggestion
$ helm install oauth2-proxy/oauth2-proxy
```

**Review by JoelSpeed — APPROVED** [review]
I think this is good, @SeWieland could you just confirm you're happy with this as well

**Review by SeWieland — APPROVED** [review]
Looks fine! :+1:

**Comment by JoelSpeed** [comment]
The release failed for this? Did we need it to succeed? Can we make sure that only real chart changes trigger the release action perhaps? (Run only if changes in the helm dir and not md files?)

**Pull request merged**